src/components/Navbar.jsx

```diff
@@ -23,12 +23,14 @@ function Navbar() {
       <div>
         <Link to="/home" className="button-link">
           <div className="navbar__icon__group">
+            <strong data-testid="home"/>
```

Comment:
what are these in a strong tag? I think you can just add it to the div. Also make the id's a bit more specific, e.g `nav-home-icon` or something like that

Reply:
Ah, the strong tag seems to be unnecessary. The tutorial I followed simply used it. I'll change that make the id's more specific!